### ISSUE TITLE
Allowed to create duplicate search scopes

### ISSUE THREAD

**Issue by poojaj-tech** [sourcegraph/sourcegraph#13082]
- **Sourcegraph Cloud** 

<img width="860" alt="Screen Shot 2020-08-17 at 3 52 09 PM" src="https://user-images.githubusercontent.com/57463526/90452385-d882dc80-e0a2-11ea-9c14-b77eb0378eb5.png">

#### Steps to reproduce:
1. Created many duplicate search scopes

#### Expected behavior:
1. I shouldn't be allowed to create multiple search scopes with the same name/title or same search query

#### Actual behavior:
1. Created many duplicate search scopes

**Comment by lguychard** [comment]
@limitedmage since "snippets" are still backed by `search.scopes`, I imagine this could still be an issue?

**Comment by limitedmage** [comment]
Yes, this should still happen, but not sure if this is a problem. Search scopes/snippets are a super low priority feature as-is.

**Comment by camdencheek** [comment]
Closing this as done. Search scopes no longer exist, so it's no longer possible to create duplicates
